.tekton/main-pipeline.yaml

```diff
@@ -301,6 +301,8 @@ spec:
     - input: $(tasks.init.results.build)
       operator: in
       values: [ "true" ]
+    # See comment in operator-bundle-build.yaml for spec.timeouts; adjustments of this timeout
+    # here might might require tweaking the timeout for the operator bundle as well.
     timeout: 2h0m0s
```

Comment:
Almost right. The place to check is `main-build.yaml` file. There's `timeouts` section with even higher value and the comment should go there.

It is simpler to orient ourselves at the pipeline timeout rather than at the build step timeout because there are several steps preceding `build-*` ones which also take time to execute. It becomes rather complex to see what's going to be the worst case of executing these steps but we know that this time is bounded by `timeouts.tasks` at the PipelineRun level.